modules/cnf-associating-secondary-interfaces-metrics-to-network-attachments.adoc

```diff
@@ -4,15 +4,15 @@
 // *networking/associating-secondary-interfaces-metrics-to-network-attachments.adoc
 
 [id="cnf-associating-secondary-interfaces-metrics-to-network-attachments_{context}"]
-= Associating secondary interfaces metrics to network attachments
+= Extending secondary network metrics for monitoring
 
 Secondary devices, or interfaces, are used for different purposes. It is important to have a way to classify them to be able to aggregate the metrics for secondary devices with the same classification.
 
-Exposed metrics contain the interface but do not specify where the interface originates. This is workable when there are no additional interfaces, but if a secondary interface is added, it is difficult to make use of the metrics since it is hard to identify the interfaces using only the interface name as an identifier.
+Exposed metrics contain the interface but do not specify where the interface originates. This is workable when there are no additional interfaces, but if a secondary interface is added, it is difficult to make use of the metrics since it is hard to identify the interfaces using only interface names.
```

Comment:
```suggestion
Exposed metrics contain the interface but do not specify where the interface originates. This is workable when there are no additional interfaces. However, if a secondary interface is added, it is difficult to use the metrics since it is hard to identify the interfaces using only interface names.
```

Reply:
Tried to break that sentence up a little. You could do "interfaces; however, if a..." as well.